draft-ietf-tls-tls13.md

```diff
@@ -4936,9 +4960,7 @@ Where Secret is either the early_exporter_master_secret or the
 exporter_master_secret.  Implementations MUST use the exporter_master_secret unless
 explicitly specified by the application. The early_exporter_master_secret is
 defined for use in settings where an exporter is needed for 0-RTT data.
-A separate interface for the early exporter is RECOMMENDED, especially
-on a server where a single interface can make the early exporter
-inaccessible.
+A separate interface for the early exporter is RECOMMENDED.
```

Comment:
I don't understand what the removed clause was trying to say.

Reply:
Oh, if a server accepts a ClientHello and sends its flight as part of processing that, then it has the full exporter immediately available.  A single exporter interface would return the full exporter.

Reply:
Ah. I will propose a rewording.

Reply:
I actually am concerned about the opposite mistake :)